scripts/apps/search/components/search-panel-widgets.tsx

```diff
@@ -12,7 +12,7 @@ export class SearchPanelWidgets extends React.PureComponent<IProps> {
     render() {
         const widgets = flatMap(
             Object.values(extensions)
-                .map((extension) => extension.activationResult?.contributions?.searchPanelWidgets ?? [])
+                .map((extension) => extension.activationResult?.contributions?.searchPanelWidgets || [])
```

Comment:
How is this a lint fix? Out of curiosity, I've checked it out the commit before this one and `npm run lint` passes without any errors.

Reply:
npm run lint did pass, but not `grunt eslint:app` which we use on travis

Reply:
Right. I now remember that I had the same issue myself and solved it by removing `eslint:app` task. Previously, 2 lint tasks were always being executed - `eslint:app` and `npm run lint`.